### ISSUE TITLE
[#15] harnessing_neural_ne_Diamond (DOI: 10.1021/acs.jpcc.3c02029)

### ISSUE THREAD

**Issue by deyulu** [yuxi120407/SpectralBench#8]
**Scenario #:** 15
**Scenario ID:** `harnessing_neural_ne_Diamond`
**DOI:** 10.1021/acs.jpcc.3c02029
**Element/Edge:** C
**Category:** synthesis

**Issue:**
21.9% sp, 24.4% sp2, and 52.8% sp3 carbon. Synthesis condition may be included in Ref. 64.

**Comment by yuxi120407** [comment]
**Fully fixed** — downloaded Ref. 64 (Osswald et al., *JACS* 2006, 128, 11635) and pipeline extracted the actual synthesis recipe.                                                                                                             
                                              
  **Note on the fractions**: your reported values (sp=0.219, sp2=0.244, sp3=0.528) sum to 0.991. The pipeline auto-normalizes to sum exactly to 1.0, giving sp=0.221, sp2=0.246, sp3=0.533. Within measurement uncertainty these are identical.  
                                                                                                                                                                                                                                                 
  **Updated output for Diamond:**                                                                                                                                                                                                                
                                                                                                                                                                                                                                                 
  Full prompt (Sample Information + Synthesis / Preparation sections):                                                                                                                                                                           
  Sample Information                                                                                                                                                                                                                             
                                                                                                                                                                                                                                               
  - Material: Nanodiamond powder (detonation soot and acid-purified samples)                                                                                                                                                                     
  - Application/Context: Evaluating global sp:sp2:sp3 hybridization ratios directly                                                                                                                                                              
                   from XANES spectra to capture complex structural distortions.                                                                                                                                                                 
                                                                                                                                                                                                                                                 
  Synthesis / Preparation                                                                                                                                                                                                                        
                                                                                                                                                                                                                                                 
  - Method:       Detonation synthesis and thermal oxidation                                                                                                                                                                                     
  - Precursors:   Carbon-containing explosives (TNT and hexogen)                                                                                                                                                                                 
  - Heating rate: 50 °C/min                                                                                                                                                                                                                      
  - Temperature:  425 °C                                                                                                                                                                                                                         
  - Duration:     5 hours                                                                                                                                                                                                                        
  - Atmosphere:   Ambient air at atmospheric pressure                                                                                                                                                                                            
  - Post-treatment: Multistage acidic purifications using nitric and sulfuric acids                                                                                                                                                              
              (prior to oxidation)                                                                                                                                                                                                               
                                                                                                                                                                                                                                                 
  **Ground truth:**
  ```json                                                                                                                                                                                                                                        
  {
    "material": "Nanodiamond powder",                                                                                                                                                                                                            
    "fit_method": "Neural network model trained on DFT-calculated spectra",                                                                                                                                                                    
    "fit_basis": ["DFT-calculated spectra of amorphous carbon"],
    "fractions": {"sp carbon": 0.221, "sp2 carbon": 0.246, "sp3 carbon": 0.533},
    "fractions_uncertainty_pct": 10
  }
